.github/workflows/docker.yml

```diff
@@ -7,15 +7,27 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true # Use "Stack CI" for all errors instead.
```

Comment:
I'd rather not set fail-fast here (or rather set it to false, as it's true by default IIRC). This basically says "if windows build fails, cancel linux build and vice versa". But those are quite different from one another, and there are many places where either could fail independently. So IMO it's more useful to always have both run to completion.